vm/main.go

```diff
@@ -34,9 +34,11 @@ func main() {
 	}
 	router.Listener = ln
 
+	ttyd = TTYD{}
+
 	router.GET("/ready", ready)
-	router.GET("/dark", dark)
-	router.GET("/light", light)
+	router.POST("/dark", dark)
+	router.POST("/light", light)
```

Comment:
IMO POST makes more sense here as it is a request that is not idempotent, a new process is started every time.